examples/react-router/app/components/index.jsx

```diff
@@ -1,22 +1,23 @@
 import ListActions from '../actions/lists'
-import ListForm    from './parts/list-form'
-import React       from 'react'
-import { Link }    from 'react-router'
+import ListForm from './parts/list-form'
+import React    from 'react'
+import {Link}   from 'react-router'
 
-const ListIndex = React.createClass({
+export default React.createClass({
```

Comment:
:facepunch: Let's discuss. There are a few trade offs with debug-ability (like components named `Index` instead of `ListIndex`, however It seems like more of the team likes this format better.